### ISSUE TITLE
Debug messages flooding error logs

### ISSUE THREAD

**Issue by JimBacon** [Indicia-Team/drupal-7-module-indicia-api#15]
If apps are left in debug mode long term then the PHP error log and, to a lesser extent, the Drupal log become so full of the debug messages that it is hard to locate errors when they occur. Look for an alternative, e.g. using a separate file for debug. When doing so, ensure the content is stored securely as it will contain sensitive information.

**Comment by kazlauskis** [comment]
Yes, it is verbous logging right now. Just want to stress the importance of it when it comes to mobile app developmet. Long term logging (up to 2-3 months) is very useful as it would be nearly imposible to debug some issues coming from user reports the problem has occurred a month ago without the logs. Plus, good to see patterns for problems :)

**Comment by JimBacon** [comment]
Sure. The point I was trying to make is that it is impeding error detection and correction of non-app problems because they are lost amongst the huge volume of debug messages coming from the apps. I'm therefore suggesting that the apps need their own log file.
